spec/factories/review.rb

```diff
@@ -10,4 +10,25 @@
       new_review_format true
     end
   end
+
+  factory :six_month_review, class: Review do
+    associate_consultant
+    review_type '6-Month'
+    feedback_deadline 6.month.from_now - 1.day
+    review_date 6.month.from_now
+  end
+
+  factory :twelve_month_review, class: Review do
+    associate_consultant
+    review_type '12-Month'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.